src/main/scala/org/apache/spark/sql/hack/SerializableConfigurationWrapper.scala

```diff
@@ -18,24 +18,18 @@
 
 package org.apache.spark.sql.hack
 
-import java.io.{ObjectInputStream, ObjectOutputStream}
-
-import org.apache.hadoop.conf.Configuration
-
-import org.apache.spark.util.Utils
+import org.apache.spark.broadcast.Broadcast
+import org.apache.spark.sql.SparkSession
+import org.apache.spark.util.SerializableConfiguration
 
 /**
- * NOTE: This is just a copy & paste of [[org.apache.spark.util.SerializableConfiguration]].
+ * This class was added because without it a NullPointerException was thrown by
```

Comment:
I'd rather leave the origin issue number with mentioning the approach we tried from Spark code, but I'll refine the comment afterwards so it doesn't matter.